mlflow/store/tracking/file_store.py

```diff
@@ -316,6 +317,16 @@ def delete_run(self, run_id):
         new_info = run_info._copy_with_overrides(lifecycle_stage=LifecycleStage.DELETED)
         self._overwrite_run_info(new_info)
 
+    def _hard_delete_run(self, run_id):
```

Comment:
Can we add documentation here explaining what this method is used for (e.g., it is used to enable the `mlflow gc` garbage collection tool and is not a public API).

Reply:
Done